changelog.md

```diff
@@ -2,7 +2,9 @@
 
 ## UNRELEASED
 
-_Nothing yet..._
+**Fixed:**
```

Comment:
Not a 100% sure how to classify this. It doesn't change anything from a cosmetical perspective, however, if someone would have used the barchart with a different component you could argue that this _might_ break because `paddingInner` is not set to 0 on the xScale.